teuthology/packaging.py

```diff
@@ -853,6 +853,9 @@ def __init__(self, project, job_config, ctx=None, remote=None):
         super(ShamanProject, self).__init__(project, job_config, ctx, remote)
         self.query_url = 'https://%s/api/' % config.shaman_host
 
+        # Force to use the "noarch" instead to build the uri.
+        self.force_noarch = self.job_config.get("force_noarch", False)
```

Comment:
Since this is related to _Shaman_ maybe use _shaman_ scope.

Reply:
Sounds good.